### PR TITLE
ARM: configs: Rebuild tegra_defconfig

### PR DESCRIPTION
**Pull request by maximschwalm**
- Rebuild tegra_defconfig
- Make the switch to the new CROS_EC options

### CODE REVIEW

**Comment by digetx** [comment]
Should be a better to fix the CONFIG_CROS_EC at first and then rebuild the config in a separate patch. The EC patch should contain the fixes tag pointing at 67c70aa86f8b796c45a08e0c955dabd826f6d0c6.

File: arch/arm/configs/tegra_defconfig [hunk 1]
```diff
@@ -118,8 +118,8 @@ CONFIG_INPUT_JOYDEV=y
 CONFIG_INPUT_EVDEV=y
 CONFIG_KEYBOARD_GPIO=y
 CONFIG_KEYBOARD_TEGRA=y
-CONFIG_KEYBOARD_CAP11XX=y
 CONFIG_KEYBOARD_CROS_EC=y
+CONFIG_KEYBOARD_CAP11XX=y
```
Comment (digetx) [inline arch/arm/configs/tegra_defconfig:116]: Shouldn't this belong to the rebuilding patch?

Comment (maximschwalm) [reply arch/arm/configs/tegra_defconfig:116]: OK, I put this change in the next commit.

**Comment by digetx** [comment]
The `fixes` tag shouldn't be wrapped, I'll correct it after merging. Thanks!

**Pull request merged**